website/src/controllers/api-v3/groups.js

```diff
@@ -303,8 +305,8 @@ api.leaveGroup = {
       }
     }
 
-    await group.leave(user, req.query.keep);
-    res.respond(200, {});
+    let leave = await group.leave(user, req.query.keep);
+    res.respond(200, leave);
```

Comment:
Just the results of group.leave - which don't seem to have to much value. We don't really assert on this ever. (Doesn't mean we shouldn't)